src/globus_sdk/exc/warnings.py

```diff
@@ -4,7 +4,13 @@
 
 
 class RemovedInV4Warning(DeprecationWarning):
-    pass
+    """
+    This warning indicates that a feature or usage was detected which will be
+    unsupported in globus-sdk version 4.
+
+    Users are encouraged to resolve these warnings when possible, so that they will be
+    able to upgrade to version 4 without issue.
```

Comment:
I don't recommend guaranteeing that resolving usage warnings now will result in zero upgrade issues in v4.

```suggestion
    Users are encouraged to resolve these warnings when possible.
```